model/init_db.js

```
@@ -38,7 +40,9 @@ function init_db() {
         .catch(function (err) {
 
 
-            // Insert dummy products
+if (products != null) {
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)